### DISCUSSION TITLE
Collapse the top part for Mobile View.

### DISCUSSION THREAD

**Discussion by bamit99** [lllyasviel/stable-diffusion-webui-forge#2071] [Ideas]
Is it possible to create a collapsable tab for the top most options as shown in the image. Would make it easy to view the UI on a phone. The top most part is not something that we change very frequently. I am ready to take a stab at it but only if someone can tell me the files I am supposed to modify.

![Screenshot_2024-10-15-14-57-55-04_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/user-attachments/assets/6208c959-53c0-41da-88c6-44ba7dbfa583)

**Comment by DenOfEquity** [comment]
You'd want to look at `modules_forge/main_entry.py` and possibly `styles.css`. High risk of breaking things.
Putting everything into a collapsible tab seems simple but gets messy - complex CSS interactions between too many elements.
One possibility is a button to toggle visibility of the elements. Not a mock up:
![togglebutton](https://github.com/user-attachments/assets/2da9aa8d-e535-41f2-b05d-f41ce00b3f73)

**Comment by bamit99** [comment]
Thanks! Let me try and break my system...Ummm I mean make it work! :D
